source/adapters/level_zero/adapter.cpp

```diff
@@ -35,19 +35,41 @@ class ur_legacy_sink : public logger::Sink {
   ~ur_legacy_sink() = default;
 };
 
-ur_result_t initPlatforms(PlatformVec &platforms) noexcept try {
+ur_result_t initPlatforms(PlatformVec &platforms,
+                          ze_result_t ZesResult) noexcept try {
   uint32_t ZeDriverCount = 0;
   ZE2UR_CALL(zeDriverGet, (&ZeDriverCount, nullptr));
   if (ZeDriverCount == 0) {
     return UR_RESULT_SUCCESS;
   }
 
+  uint32_t ZesDriverCount = 0;
+  ze_result_t zesDriverGetResult = ZE_RESULT_ERROR_UNINITIALIZED;
```

Comment:
zesinit/zesDriverGet was introduced from v1.5 onwards IIRC, so do we need a way to check version here before call zesDriverGet?